src/pycram/robot_description.py

```diff
@@ -167,6 +173,37 @@ def get_actuator_for_joint(self, joint: str) -> Optional[str]:
             return self.joint_actuators.get(joint)
         return None
 
+
+    def get_actuated_joint_names(self, part_name) -> Optional[str]:
+        """
+        Given a part name, returns the actuated joints.
+
+        Args:
+            part_name (str): part name
+
+        Returns:
+            List of actuated joint names: or None
+        """
+
+        return self.joint_actuators.get(part_name)
+
+    def get_actuated_joint_indices(self, joint_name) -> Tuple[Optional[str],Optional[int], Optional[int]]:
```

Comment:
typing

Reply:
?

Reply:
Sorry,  I meant add typing to the parameter of the function